sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -282,9 +273,9 @@ export class EventHubSender extends LinkEntity {
       }
       if (shouldReopen) {
         await defaultLock.acquire(this.senderLock, () => {
-          const options: RheaSenderOptions = this._createSenderOptions({
-            newName: true
-          });
+          const options: AwaitableSenderOptions = this._createSenderOptions(
+            Constants.defaultOperationTimeoutInMs
```

Comment:
This is where we want to recreate the sender link with new name, therefore in the old code we used to pass `newName: true` to `_createSenderOptions`

In the updated code, we are no longer asking `this. _createSenderOptions` to use the new name